File: py/PetShop.py
class PetShop:
    # atribut
    __id = ""
    __namaProduk = ""
    __hargaProduk = 0
    __stokProduk = 0
    
    # constructor
    def __init__(self, id="", namaProduk="", hargaProduk=0, stokProduk=0):
        self.__id = id
        self.__namaProduk = namaProduk
        self.__hargaProduk = hargaProduk
        self.__stokProduk = stokProduk
        
    # getter
    def getId(self): return self.__id
    def getNamaProduk(self): return self.__namaProduk
    def getHargaProduk(self): return self.__hargaProduk
    def getStokProduk(self): return self.__stokProduk
    
    # setter
    def setId(self, id): self.__id = id
    def setNamaProduk(self, namaProduk): self.__namaProduk = namaProduk
    def setHargaProduk(self, hargaProduk): self.__hargaProduk = hargaProduk
    def setStokProduk(self, stokProduk): self.__stokProduk = stokProduk

File: php/PetShop.php
<?php

class PetShop{
    // atibut
    private $id;
    private $namaProduk;
    private $hargaProduk;
    private $stokProduk;
    private $fotoProduk;

    // constructor
    public function __construct($id = '', $namaProduk = '', $hargaProduk = '', $stokProduk = '', $fotoProduk = '') {
        $this->id = $id;
        $this->namaProduk = $namaProduk;
        $this->hargaProduk = $hargaProduk;
        $this->stokProduk = $stokProduk;
        $this->fotoProduk = $fotoProduk;
    }

    // getter
    public function getId() {
        return $this->id;
    }
    public function getNamaProduk() {
        return $this->namaProduk;
    }
    public function getHargaProduk() {
        return $this->hargaProduk;
    }
    public function getStokProduk() {
        return $this->stokProduk;
    }
    public function getFotoProduk() {
        return $this->fotoProduk;
    }

    // setter
    public function setId($id) {
        $this->id = $id;
    }
    public function setNamaProduk($namaProduk) {
        $this->namaProduk = $namaProduk;
    }
    public function setHargaProduk($hargaProduk) {
        $this->hargaProduk = $hargaProduk;
    }
    public function setStokProduk($stokProduk) {
        $this->stokProduk = $stokProduk;
    }
    public function setFotoProduk($fotoProduk) {
        $this->fotoProduk = $fotoProduk;
    }

}

?>